OracleDatabase/OracleConnectionManager/dockerfile/12.2.0.1/Dockerfile

```diff
@@ -0,0 +1,87 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2018 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Connection Manager
+# 
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) linuxx64_12201_client
+#     Download Oracle Connection Manager as part of the Oracle Database Client
+#     from http://www.oracle.com/technetwork/database/enterprise-edition/downloads/index.html
```

Comment:
You probably mean http://www.oracle.com/technetwork/database/enterprise-edition/downloads/oracle12c-linux-12201-3608234.html